### PR TITLE
[ TASK-438] Added grafana support

### PR DESCRIPTION
**Pull request by Kanan99**

### CODE REVIEW

File: README.org
Comment (q-eldar) [inline README.org:1]: Move content of .org file to .md

File: monitoring/telegraf/mytelegraf.conf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Configuration for telegraf agent
 [agent]
   ## Default data collection interval for all inputs
-  interval = "20s"
+  interval = "15s"
```
Comment (q-eldar) [inline monitoring/telegraf/mytelegraf.conf:4]: Please, try decreasing interval to 10s

**Review by q-eldar — COMMENTED** [review]
Check the comments

File: monitoring/telegraf/mytelegraf.conf (hunk 1 above)
Comment (Kanan99) [reply monitoring/telegraf/mytelegraf.conf:4]: It will be decreased in monitor_persistance branch

**Pull request merged**